R/Create_metadata.R

```diff
@@ -0,0 +1,62 @@
+# find the metadata files & use them to populate the table
```

Comment:
Can we make the app run this scrip on start-up? The script has now produced App-data.RDS with absolute paths to your machine, @DrMattG .